debezium-core/src/test/java/io/debezium/transforms/outbox/EventRouterTest.java

```diff
@@ -799,9 +799,8 @@ public void canPassBinaryMessage() {
         assertThat(eventRouted).isNotNull();
         assertThat(eventRouted.keySchema().type()).isEqualTo(Schema.Type.STRING);
         assertThat(eventRouted.key()).isEqualTo(key);
-        assertThat(eventRouted.valueSchema().type()).isEqualTo(Schema.Type.STRUCT);
-        Struct valuePassed = (Struct) eventRouted.value();
-        assertThat(valuePassed.get("payload")).isEqualTo(value);
+        assertThat(eventRouted.valueSchema().type()).isEqualTo(Schema.Type.BYTES);
```

Comment:
Do we have anything to add to the change notes here? Changed assertions always raise an alert for me 🚨 ;) That said, why was is a struct before?

Reply:
@gunnarmorling I am sorry but I took a liberty and merged. The changes are in the test code introduced in DBZ-2152 and was obviously copy and past of the original one

Reply:
Ah, ok, so it was no _existing_ assert. Yeah, all cool then. Thx!

Reply:
What I find odd here is that the original PR with this test code passed Travis, so I didn't bother to run it locally (shame on me) to verify before rebasing & merging.  I understand why the test fails on master after the merging but why not prior during the PR phase is still weird.